src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -736,6 +352,13 @@ private void openWindow(List<ParserResult> loaded) {
         }
     }
 
+    /**
+     *
+     * @param name Name of the bib-file to open
+     * @param ignoreAutosave true if autosave version of the file should be ignored
+     * @return ParserResult which never is null
+     */
+
     public static ParserResult openBibFile(String name, boolean ignoreAutosave) {
```

Comment:
Could this method be extracted into a separate class, something like a AutosaveAwareParser?